_posts/2024-05-30-Improving-Dashboards-usability-with-Vega.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title:  "Improving ease of use in OpenSearch Dashboards with Vega visualizations"
+authors:
+- apasun
+- lioperry
+  date: 2024-05-30
+  categories:
+- technical-post
+  meta_keywords: dashboards, visualization, graph, vega, 
```

Comment:
Please update and use the following: 

**_meta keywords:_** OpenSearch Dashboards, Vega visualizations, visualizing data, graph